evadb/catalog/catalog_manager.py

```diff
@@ -414,9 +414,15 @@ def insert_index_catalog_entry(
         vector_store_type: VectorStoreType,
         feat_column: ColumnCatalogEntry,
         function_signature: str,
+        index_def: str,
```

Comment:
Please add the docs for func parameters or describe the need for `index_def` in the catalog file.